modules/openapi-generator/src/main/resources/python/api.mustache

```diff
@@ -65,18 +64,19 @@ class {{classname}}(object):
 {{/allParams}}
         :param async_req: Whether to execute the request asynchronously.
         :type async_req: bool, optional
-        :param _request_timeout: timeout setting for this request. If one
```

Comment:
This alignment avoids line-length issues (using `black`'s line length configuration).